skaled/main.cpp

```diff
@@ -1818,7 +1827,7 @@ int main( int argc, char** argv ) {
         }  // if --download-snapshot
 
         // download 0 snapshot if needed
-        if ( chainParams->isSyncNode() && !dataDirEmpty ) {
+        if ( chainParams->isSyncNode() && dataDirEmpty ) {
```

Comment:
I think we can keep only one of `dataDirEmpty` and `bc.number() == 0` conditions